### DISCUSSION TITLE
Any clever way to add links in Android?

### DISCUSSION THREAD

**Discussion by kanlukasz** [Kovah/LinkAce#775] [General]
Hi,

I am looking for a quick and easy way to add links while using my Android 13 phone.
Since LinkAce is not a classic Android app, there is no way to add a link through the sharing menu.

Perhaps you know of a clever way to get such a workflow? or any way with LinkAce with PWA?

I tried using [Url forwarder](https://play.google.com/store/apps/details?id=net.daverix.urlforward), but it can't open LinkAce as a PWA, so I have to log in every time in the browser (not a very good way)

**Comment by sergiorgiraldo** [comment]
I made it as a "share with Linkace". To do it, you need to add a [manifest](https://gist.github.com/sergiorgiraldo/3f50b59dc4dd837558c84315afda947c) to your site.

I made a [custom route](https://gist.github.com/sergiorgiraldo/1c97cd0a76abad77e81681bb7db7c420) to automatically get the title and link filled in the bookmarklet

**Reply by sergiorgiraldo** [reply]
I got the idea from this article: https://web.dev/articles/add-manifest

**Reply by kanlukasz** [reply]
@sergiorgiraldo can you correct the URLs?

**Reply by sergiorgiraldo** [reply]
Fixed
